moabb/datasets/bnci.py

```diff
@@ -391,29 +452,25 @@ def _convert_mi(data_paths, ch_names, ch_types):
     event_id = {}
     for filename in data_paths:
         data = loadmat(filename, struct_as_record=False, squeeze_me=True)
-        runs = []
         if type(data['data']) is np.ndarray:
             run_array = data['data']
         else:
             run_array = [data['data']]
         for run in run_array:
             raw, evd = _convert_run(run, ch_names, ch_types, None)
-            runs.append(raw) 
+            raws.append(raw)
```

Comment:
? there is no variable 'raws' in this function? Did you want to rename 'runs' to 'raws?'

Reply:
i don't remember touching this part of the code ... but good catch